R/packages.R

```diff
@@ -1,3 +1,10 @@
+SHINYLIVE_DEFAULT_MAX_FILESIZE <- "100MB"
+
+# Default maximum filesize for asset bundling
+default_max_filesize <- function() {
+  Sys.getenv("SHINYLIVE_DEFAULT_MAX_FILESIZE", SHINYLIVE_DEFAULT_MAX_FILESIZE)
+}
```

Comment:
Feels weird to have two locations that are reading in the sys envar

```suggestion
# Sys env maximum filesize for asset bundling
sys_env_max_filesize <- function() {
  Sys.getenv("SHINYLIVE_DEFAULT_MAX_FILESIZE", NULL)
}
```

Reply:
It looks like `Sys.getenv()` doesn't like `NULL` as its fallback value, it wants a character vector there.

I've gone with:
```r
sys_env_max_filesize <- function() {
  max_fs_env <- Sys.getenv("SHINYLIVE_DEFAULT_MAX_FILESIZE")
  if (max_fs_env == "") NULL else max_fs_env
}
```